.github/workflows/content-artifacts.yml

```diff
@@ -9,17 +9,14 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
   workflow_dispatch:
-    branches:
-      - main
-      - develop
 name: Process Content Artifacts
 env:
-  OSCAL_DIR_PATH: oscal
-  CICD_DIR_PATH: oscal/build/ci-cd
-  CONTENT_CONFIG_PATH: src/config
-  JAVA_CLASSPATH: ${{ github.workspace}}/lib
-  SAXON_VERSION: 9.9.0-1
-  HOME_REPO: usnistgov/oscal-content
+  HOME_REPO: usnistgov/OSCAL
+  # With the default GEN_CONTENT_DIR, the resulting catalogs, profiles, and
+  # other example content will end up in the build/generated directory. For
+  # checking in finalized content, it go into build/.., the top-level directory
```

Comment:
finalize content, it **goes** into build/

Reply:
I think we will need to meet as a team to understand this, it does by default go into `build/generated` in local development only by default, but to avoid significant churn and confusion during final deploy and to match structure of the existing repo (and not cause a really frightful commit where the whole repo gets reorganized at the same time), it goes to `build/..`, this is short-hand for "the directory above, build, AKA top-level directory of the repo." If we want further discussion, we can set this up during or after review time.

Reply:
My comment was pointing to a grammar error, not to any logistics. The grammar error is still there :
 "# checking in finalized content, **_it go into_** build/.., the top-level directory"
should be:
# checking in finalized content, **_it goes into_** build/.., the top-level directory

Reply:
Ohhh, ok. Sometimes the missing context and just the typo fix with bold alludes, especially with multiples inlcuded. My bad. That can definitely be fixed.

Reply:
Corrected in https://github.com/usnistgov/oscal-content/commit/87becb621d1be14d139b8ce41497aed11dac0164.